pmm/pmm2-release.groovy

```diff
@@ -545,12 +545,27 @@ ENDSSH
                     '''
                 }
             }
-        }        
-        stage('Run post-release tests') {
-            steps {
-                build job: 'pmm2-release-tests', propagate: false, wait: false, parameters: [
-                    string(name: 'VERSION', value: params.VERSION)
-                ]
+        }
+        stage('Run post-release tests GH Actions') {
+            steps{
+                script{
+                    withCredentials([string(credentialsId: 'GITHUB_API_TOKEN', variable: 'GITHUB_API_TOKEN')]) {
+                        sh """
+                            curl -v -X POST \
+                                -H "Accept: application/vnd.github.v3+json" \
+                                -H "Authorization: token ${GITHUB_API_TOKEN}" \
```

Comment:
There a security problem when we use variable like this inside of double quotes, i.e. sh """...""". Read more [here](https://www.jenkins.io/doc/book/pipeline/jenkinsfile/#interpolation-of-sensitive-environment-variables).

It can be fixed by using single quotes, e.g. sh '''...'''. There are good examples in other pipelines.